rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -854,20 +855,31 @@ def _make_sample_dict_from_row(bss=None, bos=None):
                 if stored_sample:
                     stored_count += 1
 
+            # PDR-243:  calculate an UNSET or FINALIZED OrderStatus to include with the biobank order data.  Aligns
```

Comment:
Note:  The "calculated" OrderStatus values are being added to both the BQBiobankOrderSchema  data (here, as `finalized_status`) and in the BQPDRBiospecimenSchema (where a previous field is being corrected to use this new `finalized_status` for its value).